src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import ToDo from '../../types/ToDo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: ToDo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, id, userId } = todo;
+
+  const foundedUser =
+    usersFromServer.find(checkedUser => checkedUser.id === userId) || '';
```

Comment:
[CODE KNOWLEDGE] - If you are using a non-mutating array method, you don't need to create a copy of the array. However, in this case, you are using the 'find' method which does not mutate the array, so you can use 'usersFromServer' directly without making a copy.